frontend/src/Authentication/Register.js

```diff
@@ -0,0 +1,150 @@
+    paddingTop: theme.spacing(3)
+  }
+}));
+
+const Register = () => {
+  const classes = useStyles();
+  
+  return (
+    <ThemeProvider theme = {theme}>
+    <TopBar/>    
+      <Box
+        display="flex"
+        flexDirection="column"
+        height="100%"
+        justifyContent="center"
+        padding = "4%"
+      >
+        <Container maxWidth="sm">
+            <Box mb={3}>
+                  <Typography
+                    color="textPrimary"
+                    variant="h3"
+                  >
+                    Create new account
+                  </Typography>
+                  <Typography
+                    color="textSecondary"
+                    gutterBottom
+                    variant="body2"
+                  >
+                    Use your email to create new account
+                  </Typography>
+                </Box>
+                <TextField
+                  fullWidth
+                  label="First name"
+                  margin="normal"
+                  name="firstName"
+                  variant="outlined"
+                />
+                <TextField
+                  fullWidth
+                  label="Last name"
+                  margin="normal"
+                  name="lastName"
+                  variant="outlined"
+                />
+                <TextField
+                  fullWidth
+                  label="Email Address"
+                  margin="normal"
+                  name="email"
+                  type="email"
+                  variant="outlined"
+                />
+                <TextField
+                  fullWidth
+                  label="Password"
+                  margin="normal"
+                  name="password"
+                  type="password"
+                  variant="outlined"
+                />
+                <Box
+                  alignItems="center"
+                  display="flex"
+                  ml={-1}
+                >
+                  <Checkbox
+                    name="policy"
+                  />
+                  <Typography
+                    color="textSecondary"
+                    variant="body1"
+                  >
+                    I have read the
+                    {' '}
+                    <Link>
+                      Terms and Conditions
+                    </Link>
+                  </Typography>
+                </Box>
+                <Box my={2}>
+                  <Button
+                    color="primary"
+                    fullWidth
+                    size="large"
+                    type="submit"
+                    variant="contained"
+                  >
+                    Sign up now
+                  </Button>
+                </Box>
+                <Typography
+                  color="textSecondary"
+                  variant="body1"
+                >
+                  Have an account?
+                  {' '}
+                  <Link component = {Link} to="/Login" >
```

Comment:
If we're using the `Link` component we don't need to add the `component` property to it as it's already of that type. This also shows up on the Login page.